test/integration/storage/test_files.rb

```diff
@@ -0,0 +1,134 @@
+require "helpers/integration_test_helper"
+
+class TestFiles < FogIntegrationTest
+  begin
+    client_email = Fog.credentials[:google_client_email]
+    @@connection = Fog::Storage::Google.new
+    @@connection.put_bucket("fog-smoke-test", options = { "acl" => [{ :entity => "user-" + client_email, :role => "OWNER" }] })
+    @@connection.put_bucket_acl("fog-smoke-test", :entity => "allUsers", :role => "READER")
+  rescue Exception => e
+    puts e
+    puts e.backtrace
+  end
+
+  begin
+    @@directory = @@connection.directories.get("fog-smoke-test")
+    @@connection.put_object("fog-smoke-test", "fog-testfile", "THISISATESTFILE")
+    @@file = @@directory.files.get("fog-testfile")
+  rescue Exception => e
+    puts e
+    puts e.backtrace
+  end
+
+  Minitest.after_run do
+    begin
+      @connection = Fog::Storage::Google.new
+      @connection.delete_object("fog-smoke-test", "fog-testfile")
+      @connection.delete_bucket("fog-smoke-test")
+    rescue Exception => e
+      puts e
+    end
+  end
+
+  def setup
+    @connection = @@connection
+    @directory = @@directory
+    @file = @@file
+  end
+
+  def test_all_files
+    skip
+  end
+
+  def test_each_files
+    skip
+  end
+
+  def test_get
+    get_file = @directory.files.get("fog-testfile")
+    assert_instance_of Fog::Storage::Google::File, get_file
+    assert_equal "THISISATESTFILE", get_file.body
+  end
+
+  def test_get_https_url
+    https_url = @directory.files.get_https_url("fog-testfile", 1000)
+    assert_match /https/, https_url
```

Comment:
Ambiguous regexp literal. Parenthesize the method arguments if it's surely a regexp literal, or add a whitespace to the right of the `/` if it should be a division.